west.yml

```diff
@@ -260,7 +260,7 @@ manifest:
       groups:
         - hal
     - name: hal_ti
-      revision: d0628446c830c25522bd1999234a77b4841016fa
+      revision: pull/68/head
```

Comment:
I don't think this is suitable mainline. You'll need to comment on that pull request and wait for it to be applied before the rest of this patch.

Reply:
I know. Once that PR is merged, we can update this PR. The pull thing seems to be a convention that allows rustbot to know which PR in other zephyr projects we need updated.